src/PAModel/PAConvert/Yaml/YamlWriter.cs

```diff
@@ -45,27 +45,39 @@ public void WriteEndObject()
             }
         }
 
-        public void WriteProperty(string propertyName, bool value)
+        public void WriteProperty(string propertyName, bool value, bool includeEquals = true)
```

Comment:
Should we make this a single function with a templated type, instead of three identical functions with one different overloaded parameter?